docs/README.mdx

```diff
@@ -1,18 +1,46 @@
+import DocCard from '@theme/DocCard';
 import DocCardList from '@theme/DocCardList';
 import {findSidebarItem} from '@site/src/sidebarUtils';
 
-# Getting Started
+# Oasis Platform Documentation
```

Comment:
This is actually just "Oasis Docs".

But perhaps not to duplicate, we could just have an inviting paragraph along the lines of:
Welcome to Oasis... This our documentation... Explore if further...

Reply:
The main title is a requirement for markdown documents. Keep in mind that this also becomes the title of the browser window/tab, so it should be something obvious to make it easy when you're switching between the tabs.

Reply:
Actually, let's revert this - the "Getting Started" seems fine since the "Oasis Documentation" is also appended to the title of each chapter.